docs/helm-workloads/deploy-helm-local.md

```diff
@@ -0,0 +1,342 @@
+
+```yaml
+replicaCount: 4
+commonLabels:
+  tier: "prod"
+```
+
+We can now deploy the prod instance:
+
+```bash
+tptctl create helm-workload-instance --config wordpress-helm-workload-instance-prod.yaml
+```
+
+Now, when we view helm workloads we can see all 3 instances derived from the
+same definition:
+
+```bash
+tptctl get helm-workloads
+```
+
+Your output should look similar to this:
+
+```bash
+NAME           HELM WORKLOAD DEFINITION     HELM WORKLOAD INSTANCE     KUBERNETES RUNTIME INSTANCE     STATUS       AGE
+wordpress      wordpress                    wordpress-default          threeport-dev-0                 Healthy      22m49s
+wordpress      wordpress                    wordpress-dev              threeport-dev-0                 Healthy      8m42s
+wordpress      wordpress                    wordpress-prod             threeport-dev-0                 Healthy      38s
+```
+
+And, again, we can see the pods from the new deployment using kubectl.
+
+```bash
+kubectl get pods -A -l control-plane.threeport.io/managed-by=threeport
+```
+
+Your output should look similar to this:
+
+```bash
+NAMESPACE                      NAME                                        READY   STATUS    RESTARTS        AGE
+wordpress-default-vpmmjfosws   wordpress-default-release-d699bdb6b-9zt4d   1/1     Running   1 (21m ago)     24m
+wordpress-default-vpmmjfosws   wordpress-default-release-d699bdb6b-fckqp   1/1     Running   0               24m
+wordpress-default-vpmmjfosws   wordpress-default-release-mariadb-0         1/1     Running   0               24m
+wordpress-dev-xsiqnjpkgd       wordpress-dev-release-d5df57df-5c8bx        1/1     Running   1 (9m23s ago)   10m
+wordpress-dev-xsiqnjpkgd       wordpress-dev-release-d5df57df-hzz46        1/1     Running   0               10m
+wordpress-dev-xsiqnjpkgd       wordpress-dev-release-mariadb-0             1/1     Running   0               10m
+wordpress-prod-fqgiwndiur      wordpress-prod-release-766467d4c-9qrv9      1/1     Running   1 (72s ago)     2m
+wordpress-prod-fqgiwndiur      wordpress-prod-release-766467d4c-n594w      1/1     Running   1 (72s ago)     2m
+wordpress-prod-fqgiwndiur      wordpress-prod-release-766467d4c-q9wp7      1/1     Running   0               2m
+wordpress-prod-fqgiwndiur      wordpress-prod-release-766467d4c-st2qn      1/1     Running   1 (72s ago)     2m
+wordpress-prod-fqgiwndiur      wordpress-prod-release-mariadb-0            1/1     Running   0               2m
+```
+
+As you can see, due to the runtime parameters for the prod instance specifying 4
+replicas, there are 4 pods for the WordPress app.
+
+Before we move on, let's clean up the helm workloads we've deployed so far.
+
+```bash
+tptctl delete helm-workload-instance -n wordpress-prod
+tptctl delete helm-workload-instance -n wordpress-dev
+tptctl delete helm-workload-instance -n wordpress-default
+tptctl delete helm-workload-definition -n wordpress
+```
+
+## Helm Workload Defined Instance
```

Comment:
Normally, I would prefer to put the easy method first.  However with Helm workloads it could be a bit confusing to understand the values docs for defs and instances.  I added a bit of an explanation at the top with a link to skip to the easy method if desired.